docs/commands.md

```diff
@@ -127,7 +126,7 @@ The data is sourced from the local cache within `.terraform` and so it may not n
    - `version` - version constraint of the module call; applicable to modules hosted by the Terraform Registry (e.g. `~> 1.0`
    - `source_type` - source of the Terraform module, e.g. `github` or `tfregistry`
    - `docs_link` - a link to the module documentation; if available
-   - `dependent_modules` - array of dependent modules with the same structure as `module_calls`
+   - `dependent_modules` - *deprecated* - array of dependent modules with the same structure as `module_calls`
```

Comment:
```suggestion
   - `dependent_modules` - **DEPRECATED** (always empty in `v0.29+`) - array of dependent modules with the same structure as `module_calls`
```

We add more context about _why_ in the Changelog, I'd just call out the version here.